crates/wasmtime/src/instance.rs

```diff
@@ -104,14 +104,15 @@ pub struct Instance {
     module: Module,
 }
 
+#[allow(dead_code)]
```

Comment:
This shouldn't be necessary.

Reply:
Otherwise it will report: warning: field is never read: `frame_info_registration`.